packager/packager.cc

```diff
@@ -500,14 +521,33 @@ class PackageInfoImpl final : public core::packages::PackageInfo {
         ENFORCE(insertionLoc.exists());
 
         auto packageToImport = info.name.toString(gs);
-        auto suggestionTitle = fmt::format("{} `{}` in package `{}`", isTestImport ? "Test Import" : "Import",
-                                           packageToImport, name.toString(gs));
+        string_view importTypeHuman;
+        string_view importTypeMethod;
+        string_view importTypeTrailing = "";
+        switch (importType) {
+            case core::packages::ImportType::Normal:
+                importTypeHuman = "Import";
+                importTypeMethod = "import";
+                break;
+            case core::packages::ImportType::TestUnit:
+                importTypeHuman = "Test Import";
+                importTypeMethod = "test_import";
+                importTypeTrailing = ", only: \"test_rb\"";
+                break;
+            case core::packages::ImportType::TestHelper:
+                importTypeHuman = "Test Import";
+                importTypeMethod = "test_import";
+                break;
+        }
+        auto suggestionTitle =
+            fmt::format("{} `{}` in package `{}`", importTypeHuman, packageToImport, name.toString(gs));
         vector<core::AutocorrectSuggestion::Edit> edits = {
-            {insertionLoc, fmt::format("\n  {} {}", isTestImport ? "test_import" : "import", packageToImport)}};
+            {insertionLoc, fmt::format("\n  {} {}{}", importTypeMethod, packageToImport, importTypeTrailing)}};
         if (deleteTestImportEdit.has_value()) {
             edits.push_back(deleteTestImportEdit.value());
-            suggestionTitle = fmt::format("Convert `{}` to `{}`", "test_import", "import");
+            suggestionTitle = fmt::format("Convert existing import to `{}`", "test_import", importTypeMethod);
```

Comment:
```suggestion
            suggestionTitle = fmt::format("Convert existing import to `{}`", importTypeMethod);
```
The title still will be odd for the case for converting a test unit import to a test helper import, but not a huge deal, we can add a TODO for it.